### DISCUSSION TITLE
24-9-10 六分街/基沃托斯/提瓦特/......每日音乐推荐（2）Beyond乐队——狂人山庄

### DISCUSSION THREAD

**Discussion by sensei-is-playing-GenShin** [share121/inter-knot#4711] [其他]
狂人山庄是Beyond乐队四子时期（对Beyond乐队历史不熟的可以看一下这个视频：【Beyond丨乐队编年简史】https://www.bilibili.com/video/BV1Xz421h7fi?vd_source=f21b5d483a76245eee948789a1043ea4） 发于1993年5月14日发行的专辑乐与怒 （在这科普一下这张专集名字的来源，因为在粤语中，“乐与怒”谐音“rock 'n'roll（应该没拼错）”）里的A面第5首歌 （磁带上就是这么标的）
![8754b229b7ea98869a2de2516bec7df5](https://github.com/user-attachments/assets/89f3595a-f280-4c67-a41f-f2d2d8430b81)同时这张专辑也是Beyond乐队四子时期的最后一张中文专辑，因为但是Beyond乐队是在日本学习，除了这张专辑外，发的都是日语专辑了。在日本后来Beyond1993年6月发生什么事大家应该都知道了就不详讲了。
最后献上网上发现的家驹照片一张
![微信图片_20240909221429](https://github.com/user-attachments/assets/2236c929-b164-4868-abf4-2a98985ac0c4)
还是老样子，如果文章中有错误的地方，请指正。如果有什么好歌都可以发到评论区的，有时间，一定会一一回复的，谢谢

**Comment by cpu644** [comment]
beyond有品的。支持继续发，对了什么时候讲到80-90s英/美摇滚。

**Comment by sensei-is-playing-GenShin** [comment]
音频链接： https://pan.baidu.com/s/1MNpjkDs3bNpwlisE3r6AvQ?pwd=qjnc 提取码: qjnc
注：有多种音质，可根据MP3或手机内存大小自行下载

**Comment by 529481** [comment]
没想到在这里也能碰到beyond的fans
